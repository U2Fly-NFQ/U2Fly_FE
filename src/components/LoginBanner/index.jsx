import { Row, Col } from 'antd'

import './style.scss'

function LoginBanner() {
  return (
    <section className="login-banner">
      <div className="grid wide">
        <Row gutter={16}>
          <Col xs={24}>
            <div className="login-banner__text">
              <h2>Log In</h2>
            </div>
          </Col>
        </Row>
      </div>
    </section>
  )
}

export default LoginBanner
