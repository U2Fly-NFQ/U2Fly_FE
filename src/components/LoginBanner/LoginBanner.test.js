import renderer from 'react-test-renderer'
import LoginBanner from '.'
import { I18nextProvider } from 'react-i18next'
import i18n from '../../translations'

describe('LoginBanner test', () => {
  it('LoginBanner component renders correctly', () => {
    const tree = renderer
      .create(
        <I18nextProvider i18n={i18n}>
          <LoginBanner />
        </I18nextProvider>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
