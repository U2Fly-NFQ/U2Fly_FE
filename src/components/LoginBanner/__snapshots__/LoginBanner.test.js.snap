// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginBanner test LoginBanner component renders correctly 1`] = `
<section
  className="login-banner"
>
  <div
    className="grid wide"
  >
    <div
      style={
        Object {
          "opacity": 0,
          "transform": "translateX(-200px) translateZ(0)",
        }
      }
    >
      <h1
        className="ant-typography"
        onClick={null}
        style={
          Object {
            "WebkitLineClamp": undefined,
          }
        }
      >
        Welcome back
      </h1>
    </div>
    <div
      style={
        Object {
          "opacity": 0,
          "transform": "translateX(200px) translateZ(0)",
        }
      }
    >
      <h3
        className="ant-typography"
        onClick={null}
        style={
          Object {
            "WebkitLineClamp": undefined,
          }
        }
      >
        Logged in to stay in touch
      </h3>
    </div>
  </div>
</section>
`;
